README.md

```diff
@@ -2,21 +2,55 @@
 <img src="https://i0.wp.com/mojaglobal.kinsta.cloud/wp-content/uploads/2021/03/Asset-66@4x.png" alt="FLINT UI logo" height ="auto" width="200" />
 <br />
 <h1>FLINT UI</h1>
-<h6>
+<p>
 A FLINT client, written in Vue, to provide an awesome user interface for configuring simulations using the FLINT.Cloud APIs
-</h6>
+</p>
 <a href="#contributors"><img src="https://img.shields.io/badge/all_contributors-1-74e8a3.svg?style=flat-square" /></a>
 <a href="https://github.com/moja-global/FLINT-UI/network/members"><img src="https://img.shields.io/github/forks/moja-global/FLINT-UI?color=74e8a3&style=flat-square" /></a>
 <a href="https://github.com/moja-global/FLINT-UI/stargazers"><img src="https://img.shields.io/github/stars/moja-global/FLINT-UI?color=74e8a3&style=flat-square" /></a>
 <a href="https://github.com/moja-global/FLINT-UI/blob/master/LICENSE"><img src="https://img.shields.io/github/license/moja-global/FLINT-UI?color=74e8a3&style=flat-square" /></a>
 </div>
 
-## About Project
+## Table of contents
+* [About Project](#about-project)
+* [Built with](#built-with)
+* [Getting started](#getting-started)
+  * [Prerequisites](#prerequisites)
+  * [Installation](#installation)
+* [Developer notes](#developer-notes)
+  * [Vue app](#vue-app)
+  * [Storybook in local development](#storybook-in-local-development)
+  * [Storybook in production](#storybook-in-production)
+  * [Dependency management](#dependency-management)
+  * [Code Style](#code-style)
+* [How to Get Involved?](#how-to-get-involved)
+* [Contributors](#contributors)
+* [Maintainers Reviewers Ambassadors Coaches](#maintainers-reviewers-ambassadors-coaches)
+* [License](#license)
```

Comment:
The Table of contents doesn't seem to be linking it as it should. Maybe you could take help of this.

![image](https://user-images.githubusercontent.com/41121520/136113794-b62300ed-fe66-4103-972a-86340941349b.png)